FORMAT.md

```diff
@@ -212,6 +212,40 @@ Note: the HRDF file is ill-formed and should generate a parsing error if both `m
 <actuator type="X5-9" mass_offset="0.2"/>
 ```
 
+### `<include>`
+
+The `include` element is used to allow commonly used snippets of HRDF to be reused in a single file or in multiple files.  The `include` element is used in place of a normal robot model element, and the filename referenced must be a complete and valid .HRDF file.  The contents of the "robot" element of this .HRDF file replace the "include" element in the final HRDF.
+
+Note that any attributes on the "robot" element in the included are ignored.  A compliant parser should generate and error if the file cannot be found.
+
+
+**Required Attributes:**
+- `path` (string) Path to the HRDF file to be included. A forward slash should be used as a file separation character. Absolute paths begin with "/", while relative paths do not.  The double dot ".." pattern moves up a directory.
+
+**Examples:**
+
+Absolute path:
+
+```xml
+<include path="/home/robotmaker/my_robot/left_arm.hrdf"/>
+```
+
+Absolute path for Windows system:
+
+```xml
+<include path="/c/users/robotmaker/my_robot/left_arm.hrdf"/>
```

Comment:
I...uh...am super unconvinced here.  I need to look at best practices for absolute windows file names with forward slashes.

Perhaps just force relative file names?

Reply:
I really don't like the `/c/` because other than MinGW I don't know anything that works like that. I don't want to reinvent a custom parser for filenames.

FWIW I think Java supports
* Windows absolute `C:\folder1\file1` and `C:/folder1/file1`
* Windows relative `.\file1` and `./file1`
* Linux/macOS absolute `/folder1/file1`
* Linux/macOS relative `./file1`

Reply:
Yeah, I agree -- I'd rather go with something already supported, as long as we can keep the config files cross-platform.

I'm fine with `c:/folder1/file1`, and that seems less ugly than `/c/folder1/file1`.  I can't think of anything besides mingw that does that either; I'm just used to it with git bash.

What about WSL -- what is its convention?

Reply:
Also, I'm fine with `./` starting for relative file names, but I don't think we should force that necessarily.  I feel like that might confuse non-unix users, especially because windows command line doesn't recognize that.